### PR TITLE
Rootage edits

### PR DESCRIPTION
**Pull request by UngluedBike**
Hope these are suitable.

### CODE REVIEW

File: rootage.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+              name: 'Hide PASELI: ****** text',
+              patches: [{offset: 0x1066CB, off: [0xF1], on: [0xF0]}]
+            },
+          ], "2019-04-18"),
+        ]);
+      });
+    </script>
+</head>
+<body>
+<h1>IIDX Cannon Ballers DLL Modder</h1>
```
Comment (Poliwrath) [inline rootage.html]: 🤔

Comment (UngluedBike) [reply rootage.html]: Ah, my mistake. Thanks for pointing it out.

Comment (UngluedBike) [reply rootage.html]: I've made the appropriate changes now.

**Comment by mon** [comment]
lgtm, cheers

**Pull request merged**